packs/kubernetes/actions/db_rds_spec.py

```diff
@@ -0,0 +1,39 @@
+import re
+import uuid
+from st2actions.runners.pythonrunner import Action
+
+
+class DatabaseRdsSpec(Action):
+    def run(self, payload):
+        # For dbidentifier
+        db_name = re.sub('[^0-9a-zA-Z]+', '-', payload['name']) + "-" + payload['namespace']
+
+        # Lets get a username generated
+        user_name = self._user_name(uid=payload['uid'])
+        # Lets get a password randomly generated
+        pw = self._id_generator()
+        instance_class = 'db.m3.xlarge'
+        port = '3306'
+        rds_engine = 'mysql'
+        allocated_storage = '50'
```

Comment:
Should be yes. I'm figuring out how to best address this. I'd like to place something like the below in config.yaml. Allowing us to rename db_rds_spec and use it as a more general piece of code for parsing through data sets.

    rds:
      size: "db.m3.xlarge"
      port: "3306"
      object: "mysql"
      storage: "100"

I'd like to use the keys above as a way to search for values in the labels. Something like payload['labels']['size']. If does not exist, use the value above as default.